src/webui/frontend/src/routes/Dashboard/Connectors.tsx

```diff
@@ -213,28 +213,61 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
         </Button>
       </BasicLayout>
     ) : (
-      <BasicLayout>
-        {connectors.map(c => (
-          <Card
-            key={c.name}
-            style={{width: 350, margin: 10, float: 'left'}}
-            actions={[
-              // eslint-disable-next-line
-              <a key={1} onClick={() => this.selectConnector(c.name)}>
-                <ApiOutlined /> Connect
-              </a>,
-            ]}
-          >
-            <Card.Meta
-              avatar={<Avatar src={`/icons/connectors/${c.name}.png`} />}
-              title={c.title}
-              description={c.description}
-              style={{height: 75}}
-            />
-          </Card>
-        ))}
-      </BasicLayout>
-    );
+        <BasicLayout>
```

Comment:
ditto here -- 2 space indent, please

Reply:
Some autoformat magic happening here. Sorry about that. Will fix now.